### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by ACPK**
Updated Mix.exs

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -6,7 +6,7 @@ defmodule PhoenixTokenAuth.Mixfile do
   def project do
     [app: :phoenix_token_auth,
      version: "0.0.14",
-     elixir: "~> 1.0",
+     elixir: "~> 1.0.4",
```
Comment (manukall) [inline mix.exs:9]: This breaks the travis-ci build. Is there a reason we can't support < 1.0.4 anymore?

**Comment by manukall** [comment]
Thanks @ACPK!

**Comment by ACPK** [comment]
@manukall  Sorry and looking into it along with another error I found. Any clue what's causing the issue below as the site gets an error when the app tries to send a confirmation email?

```
00:51:00.774 [error] GenServer Hound.SessionServer terminating
** (exit) an exception was raised:
    ** (MatchError) no match of right hand side value: {:error, %HTTPoison.Error{id: nil, reason: :econnrefused}}
        (hound) lib/hound/request_utils.ex:40: Hound.RequestUtils.send_req/4
        (hound) lib/hound/session_server.ex:22: Hound.SessionServer.handle_call/3
        (stdlib) gen_server.erl:607: :gen_server.try_handle_call/4
        (stdlib) gen_server.erl:639: :gen_server.handle_msg/5
        (stdlib) proc_lib.erl:237: :proc_lib.init_p_do_apply/3
```

**Comment by manukall** [comment]
this looks like hound can't connect to selenium of phantomjs. are they running?

**Comment by ACPK** [comment]
@manukall  - Yes but why would Hound break the site when a user signs up in the dev environment?

**Comment by manukall** [comment]
i really don't know. usually hound shouldn't even be available in the dev environment. in the example app i added it as `{:hound, ">= 0.6.0", only: :test}` in mix.exs, so it's only available in the test environment.

**Comment by ACPK** [comment]
@manukall We do too. The issue happens with my updated fork (this PR).

**Pull request closed without merging**